src/plugins/intel_gpu/src/graph/graph_optimizer/add_required_reorders.cpp

```diff
@@ -130,6 +131,22 @@ void add_required_reorders::run(program& p) {
             }
         }
 
+        // Remove padded-inputs in spatial axes not to use ref kernel which causes huge perf drop
+        if (usr->is_type<mvn>() && usr->as<mvn>().input().is_padded_spatial()) {
+            auto out_layout = usr->get_output_layout();
+            // Check formats of implemented opt kernels without a spatial padding support
+            if (out_layout.format == format::b_fs_yx_fsv16 || out_layout.format == format::b_fs_zyx_fsv16 ||
+                out_layout.format == format::bs_fs_yx_bsv32_fsv16 || out_layout.format == format::bs_fs_yx_bsv32_fsv32) {
+                auto& dep = usr->as<mvn>().input();
+                cldnn::layout layout_wo_padding = dep.get_output_layout();
+                layout_wo_padding.data_padding = cldnn::padding{};
+                auto new_reorder = std::make_shared<reorder>(dep.id() + "_no_pad_reorder", dep.id(), layout_wo_padding);
+                auto& new_reorder_node = p.get_or_create(new_reorder);
+                p.add_intermediate(new_reorder_node, *usr, dep);
+                new_reorder_node.recalc_output_layout(false);
+            }
+        }
```

Comment:
did you review to add this reorder in `reorder_inputs` instead of `add_required_reorder`?

Reply:
It is related to padding propagation by opt out of reshape and reorder. Reorder_inputs is done before  reshape handle and remove reorder. So I added this logic into add_required_reorder.